spec/models/piece_spec.rb

```diff
@@ -50,12 +50,104 @@
       expect(move.game_id).to eq(@game.id)
     end
 
-    it "should raise an error if the arguments are out of bounds" do
+    # MeO tests for move_to! - test valid_move? and its called methods
+    from_row = 3
```

Comment:
Since these never change, it's better to make them constants by upper-casing them:

```ruby
FROM_ROW = 3
FROM_COL = 2
```